File: code.py
import numpy as np
import pandas as pd

simplon = np.array([["les maths", "le SQL", "parler de servers", "Python", "chercher des stages"],
["parler d'enfants", "plus de SQL", "les pcs Shadow", "les exercices alambiqués", "les potits chiens"]])


def create_message(answer):
    if answer == 1:
        message = messages[0]
    elif answer == 2: 
        message = messages[1]
    elif answer == 3: 
        message = messages[2]
    elif answer ==4: 
        message = messages[3]
    elif answer == 5: 
        message = messages[4]
    elif answer < 1 or answer > 5:
        message = messages[5]
    else:
        message = messages[6]
    return message

# Faire un tableau des résultats possibles
messages = [ 
    "Vous aimez " + simplon[0,0] + " et aussi " + simplon[1,0],
    "Vous aimez " + simplon[0,1] + " et aussi " + simplon[1,1],
    "Vous aimez " + simplon[0,2] + " et aussi " + simplon[1,2],
    "Vous aimez " + simplon[0,3] + " et aussi " + simplon[1,3],
    "Vous aimez " + simplon[0,4] + " et aussi " + simplon[1,4],
    "Ce n'est pas un chiffre correct. Faut apprendre à lire.",
    "Allez, la bonneuh journée!"
]

# Afficher ces résultats
#print("Tu seras dans une de ses catégories :")
#for message in messages:
# print("- " + message)

answer = int(input("Qui préferez-vous? Jo = 1, Emma = 2, Arnaud = 3, Flo = 4, Perrine = 5 : \n> "))
message = create_message(answer)

print(message)

File: js/code.js
const messages = ["Vous aimez les maths et aussi parler d'enfants.", 
"Vous aimez le SQL et aussi plus de SQL.", 
"Vous aimez parler de servers et aussi les Shadows.", 
"Vous aimez Python et aussi les exercices alambiqués.", 
"Vous aimez chercher des stages et aussi les potits chiens.", 
"Ce n'est pas un chiffre correct. Faut apprendre à lire.", 
"Allez, la bonneuh journée!"];

const createMessage = (answer) => {
    let message = '';
    if (answer == 1) {
        message = messages[0];
    } else if (answer == 2) { 
        message = messages[1];
    } else if (answer == 3) { 
        message = messages[2];
    } else if (answer == 4) { 
        message = messages[3];
    } else if (answer == 5) { 
        message = messages[4];
    } else if (answer < 1) {
        message = messages[5];
    } else if (answer > 5) {
        message = messages[5];
    } else {
        message = messages[6];
    } return `Ah! ${message} `;
  };
 

const form = document.getElementById("my-form");
console.log(form);
  
const message = document.getElementById("ia-message");
const counter = document.getElementById("counter");
let count = 0;
  
form.addEventListener("submit", (event) => {
    const input = form.querySelector("#user-answer");
    answer = input.value
    message.innerHTML = createMessage(answer);
    count = count += 1
    counter.innerHTML = count;
  });
  